src/components/families/family-list/family-sidebar/FamilySidebar.tsx

```diff
@@ -159,6 +171,35 @@ const FamilySidebar = ({ family, isOpen, onClose }: Props) => {
           </Box>
         )}
 
+        <Box position="relative">
```

Comment:
There are two dividers back to back if there are no guests under a family. Don't think this issue was introduced in this PR, but just wanted to bring it up because idk how to create GitHub issues under this repo.

![Screen Shot 2021-07-27 at 10 01 03 PM](https://user-images.githubusercontent.com/21024100/127251728-80e34bde-8025-4fa3-8e6b-a2e2c466f20c.png)

Reply:
ooh yeah i was meaning to fix that :) i'll put up a fix but for future stuff feel free to make a task in notion